error  too much

➜  tars-spring-boot-server git:(v1.7.x) ✗ cd ../tars-spring-boot-client
➜  tars-spring-boot-client git:(v1.7.x) ✗ mvn package                  
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[WARNING] 'build.plugins.plugin.version' for org.springframework.boot:spring-boot-maven-plugin is missing. @ line 122, column 21
[ERROR] 'dependencies.dependency.version' for com.tencent.tars:tars-spring-boot-starter:jar must be a valid version but is '${revision}'. @ line 38, column 22
 @

➜  quickstart-server git:(v1.7.x) ✗ cd ../quickstart-client 
➜  quickstart-client git:(v1.7.x) ✗ mvn package       

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:2.4:war (default-war) on project tars-quickstart-server: Execution default-war of goal org.apache.maven.plugins:maven-war-plugin:2.4:war failed: Unable to load the mojo 'war' in the plugin 'org.apache.maven.plugins:maven-war-plugin:2.4' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: null

--- a/examples/quickstart-client/pom.xml
+++ b/examples/quickstart-client/pom.xml
@@ -7,6 +7,7 @@
     <name>test</name>
     <packaging>war</packaging>
     <properties>
+        <revision>1.7.3</revision>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
     <dependencies>

--- a/examples/quickstart-server/pom.xml
+++ b/examples/quickstart-server/pom.xml
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>com.tencent.tars</groupId>
 		<artifactId>tars-parent</artifactId>
-		<version>${revision}</version>
+		<version>1.7.3</version>
 	</parent>
 	<artifactId>tars-quickstart-server</artifactId>
 	<name>${project.artifactId}</name>
@@ -14,6 +14,7 @@
 		<mavenVersion>2.2.1</mavenVersion>
 		<mavenPluginPluginVersion>3.2</mavenPluginPluginVersion>
 		<sysout-over-slf4j.version>1.0.2</sysout-over-slf4j.version>
+		<maven_war_plugin_version>3.2.2</maven_war_plugin_version>
 	</properties>
 	<dependencies>
 		<dependency>
@@ -51,7 +52,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-war-plugin</artifactId>
-				<version>2.4</version>
+				<version>${maven_war_plugin_version}</version>
 			</plugin>
 
 			<plugin>

--- a/examples/quickstart-server/src/main/java/com/qq/tars/quickstart/server/testapp/HelloServant.java
+++ b/examples/quickstart-server/src/main/java/com/qq/tars/quickstart/server/testapp/HelloServant.java
@@ -6,6 +6,7 @@
 package com.qq.tars.quickstart.server.testapp;
 
 import com.qq.tars.protocol.annotation.Servant;
+import com.qq.tars.protocol.tars.annotation.TarsMethodParameter;
 
 @Servant
 public interface HelloServant {

